tests/managed_services/mas_debezium/test_mas_debezium.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from constants import TEST_RECORD
+
+
+pytestmark = pytest.mark.mas_debezium
+
+
+class TestDebezium:
+    def test_kafka_topics(
+        self, kafka_instance_client, kafka_topics, kafka_sa_acl, consumer_pod
+    ):
+        """
+        Test for managed kafka resources setup,
+        usage and teardown via rhoas sdk
+        """
+        # Get consumed record from test topic via kcat pod
+        kafka_event = consumer_pod.log()
```

Comment:
```suggestion
        kafka_log = consumer_pod.log()
```
log may contain more than just events

Reply:
done